src/libs/components/expanded-tile-swiper/expanded-swiper.loader.ts

```diff
@@ -60,39 +75,103 @@ function initializeSwiperForExpandedTiles(initialTileId: string) {
 function controlOnLoadPlayback() {
   const swiper = getInstance("expanded")
   if (swiper) {
-    const activeElement = getSwiperVideoElement(swiper, swiper.realIndex)
-    activeElement?.play()
+    const activeElementData = getSwiperVideoElement(swiper, swiper.realIndex)
+    triggerPlay(activeElementData)
   }
 }
 
 function controlVideoPlayback(swiper: Swiper) {
   const activeElement = getSwiperVideoElement(swiper, swiper.realIndex)
   const previousElement = getSwiperVideoElement(swiper, swiper.previousIndex)
 
-  activeElement?.play()
+  // play video control
```

Comment:
Redundant comments

Reply:
I am not sure these are redundant. One is play and the other one is pause. I will maybe remove it and add documentation for the methods